cmd/sebak/cmd/key/verify

```diff
@@ -0,0 +1,131 @@
+package key
```

Comment:
@soonkuk What is this? Is it related to this PR?

Reply:
@spikeekips Oh it's my mistake. I'll remove that file.